tests/unit/test_utils.py

```diff
@@ -26,7 +26,7 @@ def test_load_json_succes(self, tmp_file):
         assert config['type'] == 'web'
         assert config['devices'] == ['nexus6p']
         assert config['randomization'] == 'False'
```

Comment:
Should be `false` instead of `'false'`?

Reply:
Good catch.  I searched for any occurrence of random or randomization to see if there was anything left to update.  My latest commit is reflective of that.